src/components/CategoryEditTree.vue

```diff
@@ -57,7 +57,7 @@ import 'vue-awesome/icons/edit';
 export default {
   name: 'CategoryEditTree',
   props: {
-    cls: Object,
+    _class: Object,
```

Comment:
Why the `cls` -> `_class` rename? 

`cls` is commonly used in Python (due to it being a reserved keyword). I'd personally prefer to not have underscores in names (unless they are private/local/special), but that's just me.

Reply:
Because I googled for what cls might mean but couldn't find what it referred to (clear screen in DOS, common language specification, etc. were the results).
I understood it after a while, but it took me way too long.
I've seen klass and _class before so I went with _class.

Reply:
Fair enough, I guess it's not that clear except in the Python-context:

```py
class A:
    @classmethod
    def f(cls, *args):
        return cls(*args)
```